flashinfer/gemm/routergemm.py

```
@@ -400,6 +406,68 @@ def tinygemm2_nobias_op_impl(
+
+
+@functools.cache
+def get_tinygemm2_sm100_module():
+    module = gen_tinygemm2_sm100_module().build_and_load()
+
+    @register_custom_op(
+        "flashinfer::tinygemm2_sm100_op",
+        mutates_args=["out"],
+    )
+    def tinygemm2_sm100_op_impl(
+        input: torch.Tensor,
```

Ruff (0.16.0)

[error] Function argument `input` is shadowing a Python builtin

(A002)